src/AmplNLWriter.jl

```diff
@@ -236,7 +111,7 @@ Create a new Optimizer object.
   and then destructs the environment.
 
 `solver_args` is a vector of `String` arguments passed solver executable.
-However, prefer passing `key=value` options via `MOI.RawParameter`.
+However, prefer passing `key=value` options via `MOI.RawOptimizerAttribute`.
 
 Redirect IO using `stdin` and `stdout`. These arguments are passed to
 `Base.pipeline`. See the Julia documentation for more details.
```

Comment:
It might be useful to give a link to the exact part of the docs (especially as `pipeline` has two functions). I think https://docs.julialang.org/en/v1/base/base/#Base.pipeline-Tuple{Base.AbstractCmd} is the best link.